com.playeveryware.eos/Runtime/Core/Config/Config.cs

```diff
@@ -208,6 +218,11 @@ public static async Task<T> GetAsync<T>() where T : Config
             // Use the factory method to create the config.
             T instance = (T)factory();
 
+            if (!await FileSystemUtility.FileExistsAsync(instance.FilePath) && instance._allowDefaultIfFileNotFound)
```

Comment:
Will this break in Android?

Reply:
It will not break in Android because `GetAsync` itself is not available in Android.